### ISSUE TITLE
Use CollectionFS with cfs:filesystem in multiple instances?

### ISSUE THREAD

**Issue by bluepuma77** [CollectionFS/Meteor-CollectionFS#982]
I am running CollectionFS with cfs:filesystem on multiple instances with the same code.
- 1 instance is only handling the upload and the processing
- 2+ instances are only handling image serving

When generating lots of files with instance 1 I experience other instances crashing:

```
W20160629-18:15:00.618(0)? (STDERR) stream.js:94
W20160629-18:15:00.618(0)? (STDERR)       throw er; // Unhandled stream error in pipe.
W20160629-18:15:00.619(0)? (STDERR)             ^
W20160629-18:15:00.619(0)? (STDERR) Error: ENOENT, open '/app/.meteor/local/cfs/files/_tempstore/files-fhftx3PHM7cRwKHBa-0.chunk'
=> Exited with code: 8
=> Your application is crashing. Waiting for file change.
```

I read through some posts, seems CollectionFS is not made for multiple instances with the same database and file system because of the current design with the worker.

Two questions from my side:
1. Can I just disable the workers on the file-serving instances?
2. Can I just try/catch the issue? Has someone done this before?

**Comment by jeffyamada** [comment]
@bluepuma77 Yeah, we're having the same issue - and were running multiple dynos on heroku. Did you find anything that worked?

**Comment by rolljee** [comment]
Any update ?

**Comment by evolross** [comment]
Experiencing the same problem.  I've looked all over and I don't think an official fix was ever made. `edgee:slingshot` anyone?
